tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+from pathlib import Path
+
+import pytest
+
+import ecl2df
+
+
+@pytest.fixture
+def path_to_ecl2df():
+    """Path to installed ecl2df module"""
+    return Path(ecl2df.__file__).parent
```

Comment:
Should we consider absolute path?

Reply:
`__file__` is (almost) always absolute, since Python 3.4 it seems.